docs/connector.rst

```diff
@@ -73,6 +73,16 @@ Once you have created a ``Connector`` instance, :meth:`Connector.get_output()`
 returns the :class:`Output` that allows writing data, and :meth:`Connector.get_input()`
 returns the :class:`Input` that allows reading data.
 
+.. note::
+
+  If the *domain_participant* you load contains both *data_writers* (Outputs) and
+  *data_readers* (Inputs) for the same topics, when you write data, the Inputs
```

Comment:
for the same topics -> which match (are associated with the same topic and have compatible QoS)